Daytime-Nighttime-Interactions/fault-reporting.rst

```diff
@@ -41,10 +41,11 @@ Some guidelines to keep in mind are:
 
 - Facts first. 
   The author of the fault report should provide as many details as possible, including screenshots, telescope telemetry, and timestamps for future investigation.
-- Leave the ticket unassigned, it will be triaged in the morning or self-assigned. 
-  If observers think people need to be aware of the ticket (e.g. prospective assignees), they should be @'d in a comment
+- Leave the ticket unassigned, it will be triaged by the traige team, whos membership includes T&S and SIT-Com members. 
+  If observers think people need to be aware of the ticket (e.g. prospective assignees), they should be @'d in a comment.
```

Comment:
People can also be included as watcher to the ticket. This might be a better way to broadcasting the ticket than to @-them no?

Reply:
👍